### ISSUE TITLE
项目是否支持1080Ti的卡

### ISSUE THREAD

**Issue by liutianling** [WongKinYiu/ScaledYOLOv4#141]
这个项目真的非常棒，谢谢分享。
我这边遇到一个问题，希望得到您的帮助。
我在Tesla T4 的卡上，跑咱们的项目，模型表现的非常好。
但是当我在1080Ti的卡上训练模型时，出现了这个问题：

GIou obj cls 都是nan，

 Epoch   gpu_mem      GIoU       obj       cls     total   targets  img_size
      0/99     8.08G       nan       nan       nan       nan         9       640: 100% 4/4 [00:12<00:00,  3.11s/it]
/usr/local/lib/python3.6/site-packages/torch/optim/lr_scheduler.py:123: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)
****************************************************************************************************
               Class      Images     Targets           P           R      mAP@.5  mAP@.5:.95: 100% 1/1 [00:01<00:00,  1.98s/it]
                 all           7           0           0           0           0           0

     Epoch   gpu_mem      GIoU       obj       cls     total   targets  img_size
      1/99     9.34G       nan       nan       nan       nan         3       640: 100% 4/4 [00:05<00:00,  1.26s/it]
****************************************************************************************************
               Class      Images     Targets           P           R      mAP@.5  mAP@.5:.95: 100% 1/1 [00:00<00:00,  1.19it/s]
                 all           7           0           0           0           0           0

     Epoch   gpu_mem      GIoU       obj       cls     total   targets  img_size
      2/99     9.34G       nan       nan       nan       nan         5       640: 100% 4/4 [00:04<00:00,  1.20s/it]
****************************************************************************************************
               Class      Images     Targets           P           R      mAP@.5  mAP@.5:.95: 100% 1/1 [00:01<00:00,  1.64s/it]
                 all           7           7     0.00166        0.25     0.00219    0.000655

     Epoch   gpu_mem      GIoU       obj       cls     total   targets  img_size
      3/99     9.34G       nan       nan       nan       nan         5       640: 100% 4/4 [00:05<00:00,  1.31s/it]
****************************************************************************************************
               Class      Images     Targets           P           R      mAP@.5  mAP@.5:.95: 100% 1/1 [00:00<00:00,  1.18it/s]
                 all           7           0           0           0           0           0

**Comment by WongKinYiu** [comment]
您好
我自己是用 titanX (Maxwell)開發, 沒什麼問題。
您是否有使用到提供的 pre-trained weights ?
若有的話可能會有些 half 和 float 的轉換問題。
我尚未測試把同一個 model 放在不同機器上跑。

**Comment by liutianling** [comment]
非常感谢您的及时回复。
有用到提供的yolov4-p6.pt 模型。
我看下转换那部分的代码，谢谢您！

**Comment by liutianling** [comment]
再请假下torch版本的安装问题。
您提供的这个镜像 nvcr.io/nvidia/pytorch:20.06-py3 中，
torch 的版本为 1.6.0a0+9907a3e， torchvision的版本为 0.7.0a0
这种版本的该如何安装， 直接pip install 找不到对应的版本
@WongKinYiu 谢谢！

**Comment by WongKinYiu** [comment]
https://github.com/pytorch/pytorch/commit/b0b9e704ed1f31ed1c81994c20e5a9f3715d2583

**Comment by liutianling** [comment]
非常感谢 @WongKinYiu
